help.c

```diff
@@ -107,6 +107,16 @@ Values:\n\
     0b...   Binary number\n\
     0y...   Binary number\n\
     ...     Decimal number\n\
+\n\
+Suffixes (for common memory sizes):\n\
+    ...K, ...k, ...KiB    Kilobyte (1024 bytes)\n\
```

Comment:
the "iB" suffixes are "ibibytes".  so this is Kibibyte, not kilobyte.

rather than have to document every one explicitly, we could do something like:

```
     ...K, ...k, ...KiB    Kibibyte (1024 bytes)\n\
     ...KB                 Kilobyte (1000 bytes)\n\
     M/G/T/P/E/Z/Y suffixes are similarly supported.\n\
```